core/src/test/java/org/infinispan/expiration/impl/ExpirationStoreListenerFunctionalTest.java

```diff
@@ -14,13 +14,21 @@
 import org.infinispan.persistence.spi.AdvancedCacheExpirationWriter;
 import org.infinispan.test.TestingUtil;
 import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Factory;
 import org.testng.annotations.Test;
 
 @Test(groups = "functional", testName = "expiration.impl.ExpirationStoreListenerFunctionalTest")
 public class ExpirationStoreListenerFunctionalTest extends ExpirationStoreFunctionalTest {
    protected ExpiredCacheListener listener = new ExpiredCacheListener();
    protected ExpirationManager manager;
 
+   @Factory
+   public Object[] factory() {
+      return new Object[]{
+            new ExpirationStoreListenerFunctionalTest(),
```

Comment:
Missing `@Override` and maybe a comment to explain why we don't need super's parameters.

Reply:
Sure, let me fix it.